README.md

```diff
@@ -80,15 +81,21 @@ You can setup storages either at zenoh router startup via a configuration file,
               volume: {
                 id: "influxdb",
                 // the database name within InfluxDB
+                // NOTE: For influxdb v2.x, the influxdb rust library currently doesn't provide for creating the database so this db should already exist
```

Comment:
Same comment as above.
Weird wording... *doesn't provide for*? Did you mean *doesn't provide SUPPORT for*?